# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

InteractiveBlockManipulationActionGoal action_goal
InteractiveBlockManipulationActionResult action_result
InteractiveBlockManipulationActionFeedback action_feedback
